### DISCUSSION TITLE
add some "little" features :)

### DISCUSSION THREAD

**Discussion by kenyue102008** [DustinBrett/daedalOS#236] [Feature Requests]
Hi, can you please add a settings app, so we can customize the pc username, and the most important is customize the start menu button icon, we can customize it with our own picture from local pc through settings. Also, adding search button and task view would be nice :3 Thank you so much for reading this, I hope to see these features in daedalos, my favorite browser os :)

**Comment by DustinBrett** [comment]
Thanks for the suggestions/requests! Search button and Task View are indeed planned for 2023. As for changing the name or start menu icon, those things are possible, in the case of the name it's a bit harder, but likely those features won't come until I create customization dialogs which I am also planning. Thanks for the notes/support, I will keep all this in mind and add it to my lists.
